.eslintrc.json

```diff
@@ -26,6 +26,7 @@
   "parserOptions": {
     "ecmaVersion": 2018,
     "sourceType": "module",
+    "project": "./tsconfig.json",
```

Comment:
curious, if it's the default, no? I didn't know this makes any difference.

Reply:
Probably it's a default. Set it to be 100% sure. It used to be required I think.

Reply:
Probably it's a default. Set it to be 100% sure. It used to be required I think.